livetest/livetest.sh

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/env bash
+
+# TODO: Env vars for argument and configured values for the Kafka Cluster
+
+# Deploy kafka client to use as producer and consumer
+# TODO: figure out if we can deploy client without topic and users first?
+kubectl apply -n kafka -f kafka-topics.yaml
+kubectl apply -n kafka -f kafka-users.yaml
+kubectl apply -n kafka -f kafka-client.yaml
+sleep 5s
+
+# Kafka client SSL Components
+setup_kafka_client_ssl () {
```

Comment:
@mtarng we don't seem to be using this. Especially this this test is using `kubectl exec` We can probably get rid of this method and reference until we have a test that using ingress